src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -1051,6 +1052,14 @@
                 "enum": ["kebabCase"]
               }
             }
+          },
```

Comment:
Thanks for adding this! Post-PR can you also update the canonical schema at schemastore?

Reply:
Absolutely. Is there some documentation on this process that I can read up on? I have not contributed here before so want to make sure I am doing it correctly.

Reply:
There's no heavy process here - it's just that many tools (including VSCode) integrate easily with Schemastore so we also have the schema mirrored there at https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/template.json - you can submit changes to it same as you did here and tag me on the PR and I'll approve it. The one here in the repo is the 'most correct' one, but the one in Schemastore is the most tooling-friendly one.

Reply:
@baronfel I opened https://github.com/SchemaStore/schemastore/pull/4230